optimade/server/query_params.py

```diff
@@ -169,9 +199,12 @@ def __init__(
         self.page_above = page_above
         self.page_below = page_below
         self.include = include
+        self.first_frame = first_frame
+        self.last_frame = last_frame
+        self.frame_step = frame_step
```

Comment:
Following my comment above, we probably need some endpoint-specific query parameters now

Reply:
Setting these parameters for a structure or other endpoint would not make sense, and they would simply be ignored. So I do not see why we would need to complicate things by creating entry point specific query parameters.

Reply:
Following my comment above, I would argue that it complicates the *other* parts of the spec by including them. None of the existing implementations will support this parameter, but it will be added to the spec here in a very general way.